sleap/gui/app.py

```diff
@@ -1012,55 +1014,96 @@ def _add_button(to, label, action, key=None):
 
-        skeletons_folder = sleap.util.get_package_file("sleap/skeletons")
-        skeletons_json_files = sleap.util.find_files_by_suffix(
+        skeletons_folder = get_package_file("sleap/skeletons")
+        skeletons_json_files = find_files_by_suffix(
             skeletons_folder, suffix=".json", depth=1
         )
         skeletons_names = [json.name.split(".")[0] for json in skeletons_json_files]
         skeletons_names.insert(0, "Custom")
-        hb.addWidget(QLabel("Load from:"))
+        hb.addWidget(QLabel("Load from preset:"))
         self.skeletonTemplates = QComboBox()
         self.skeletonTemplates.addItems(skeletons_names)
         self.skeletonTemplates.setEditable(False)
         hb.addWidget(self.skeletonTemplates)
         hb.addWidget(QLabel("skeleton template"))
-
         hbw = QWidget()
         hbw.setLayout(hb)
+        vb.addWidget(hbw)
+
+        hb = QHBoxLayout()
+        preview_image = QtGui.QPixmap(get_package_file("sleap/gui/no-preview.png"))
+        self.skeleton_preview_image = QLabel("Preview Skeleton")
+        self.skeleton_preview_image.setPixmap(preview_image)
+        hb.addWidget(self.skeleton_preview_image)
+        hb.setAlignment(self.skeleton_preview_image, Qt.AlignLeft)
+
         self.skeleton_description = QLabel(
-            f'Description: {self.state["skeleton_description"]}'
+            f'<strong>Description:</strong> {self.state["skeleton_description"]}'
         )
-        vb.addWidget(self.skeleton_description)
+        self.skeleton_description.setWordWrap(True)
+        hb.addWidget(self.skeleton_description)
+        hb.setAlignment(self.skeleton_description, Qt.AlignLeft)
+
+        hbw = QWidget()
+        hbw.setLayout(hb)
+        vb.addWidget(hbw)
 
         self.state.connect(
             "skeleton_description",
-            lambda label: self.skeleton_description.setText(f"Description: {label}"),
+            lambda label: self.skeleton_description.setText(
+                f"<strong>Description:</strong>: {label}"
+            ),
         )
 
         def updatePreviewImage(preview_image: bytes):
             if preview_image is None:
-                print("No preview image available to display!")
                 preview_image = QtGui.QPixmap(
-                    sleap.util.get_package_file("sleap/gui/no-preview.png")
+                    get_package_file("sleap/gui/no-preview.png")
                 )
             else:
-                preview_image = sleap.util.decode_preview_image(preview_image)
-                preview_image = ImageQt(preview_image)
+
+                preview_image = decode_preview_image(preview_image)
+
+                # Create a QImage from the Image
+                preview_image = QtGui.QImage(
+                    preview_image.tobytes(),
+                    preview_image.size[0],
+                    preview_image.size[1],
+                    QtGui.QImage.Format_RGB888,  # Format for RGB images (see Image.mode)
+                )
+
```

Comment:
Before (stochastic bug)
---
Using `ImageQt` to convert our image to a `QImage` would result in stochastic incorrect display of the `Pil.Image`s (which were displayed perfectly with `Image.show()`):
![image](https://user-images.githubusercontent.com/38435167/217958947-a674c7a3-c8e0-4bf3-b6c4-5a37796a22ff.png)

Current (deterministic bug)
---
I changed this to manually convert the `PIL.Image` to a `QImage` which ALWAYS displays MOST images fine, execept for one which always shows up as seen below:
![image](https://user-images.githubusercontent.com/38435167/217959230-fb26acd0-ea8b-46a5-ae7d-0374d44ac8ee.png)